### PR TITLE
First implementation of win32 window

### PR DESCRIPTION
**Pull request by jonathanudd**
First implementation of win32 window with the ability to draw images without a GPU.

### CODE REVIEW

File: source/ui/win32/Win32Window.ooc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software. If not, see <http://www.gnu.org/licenses/>.
+ */
```
Comment (davidhesselbom) [inline source/ui/win32/Win32Window.ooc]: Add an empty line here.

Comment (jonathanudd) [inline source/ui/win32/Win32Window.ooc]: Fixed this.

File: source/ui/win32/include/win32.ooc [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+HICON: cover from Void*
+HCURSOR: cover from Void*
+HBITMAP: cover from Void*
+HDC: cover from Void*
+HGDIOBJ: cover from Void*
+
+PAINTSTRUCT_OOC: cover from PAINTSTRUCT
+POINT_OOC: cover from POINT
+RECT_OOC: cover from RECT
+MSG_OOC: cover from MSG {
```
Comment (davidhesselbom) [inline source/ui/win32/include/win32.ooc]: If you want to keep the same name as the original struct, use `PAINTSTRUCT: extern cover`.

If you want to give them your own names, then give them proper ooc names: `PaintStruct: cover from PAINTSTRUCT`.

Comment (davidhesselbom) [inline source/ui/win32/include/win32.ooc]: My point is, ugly names are not avoidable within the wrapper, but I do not want to see things like `HWND` anywhere outside this file. See https://ooc-lang.org/docs/lang/covers/ and do your best to give functions, variables, constants and anything everything else proper ooc names.

**Comment by fredrikbryntesson** [comment]
Magic fails this.

**Comment by jonathanudd** [comment]
Fixed magic errors.

**Comment by fredrikbryntesson** [comment]
Build and run on both pc and android, memcheck ok and magic ok.

**Pull request merged**